index.html

```diff
@@ -0,0 +1,134 @@
+            <p class="paragraph">I'm a software developer!I can help you build a product, feature or website. Look through some of my work and experience! If you like waht you see and have a project you need coded, dont hesitate to contact me.</p>
+            <div class="connect">LET'S CONNECT</div>
+            <ul id="social-links">
+                <li><a href="#"></a><img src="Assets/files/assets/twitter.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/Linkedin icon.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/medium.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/github.png" alt=""></li>
+                <li><a href="#"></a><img src="Assets/files/assets/Vector.jpg" alt=""></li>
+            </ul>
+        </section>
+    </section>
+    <section>
+        <ul  id="works">
+            <li class="container1">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot.png" alt=""></li>
+                    <li class="works-heading">Tonic</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+            <li class="container2">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot2.png" alt=""></li>
+                    <li class="works-heading">Multi-Post Stories</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
+                    <li>
+                        <ul class="languages">
+                            <li>HTML</li>
+                            <li>CSS</li>
+                            <li>Javascript</li>
+                        </ul>
+                    </li>
+                    <li><button>See project</button></li>
+                </ul>
+            </li>
+            <li class="container1">
+                <ul>
+                    <li class="works-pic"><img src="Assets/files/Snapshoot3.png" alt=""></li>
+                    <li class="works-heading">Tonic</li>
+                    <li class="canopy"><img src="Assets/files/Frame 2.png" alt=""></li>
+                    <li class="works-paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required.</li>
```

Comment:
- According to the project requirements, please use a `p` tag, it'd help improve the semantics of your site. Please fix this for all project cards🙏